spec/controllers/line_items_controller_spec.rb

```diff
@@ -60,8 +61,8 @@
         context "without an order cycle" do
           it "denies deletion" do
             delete :destroy, params
-            expect(response.status).to eq 403
-            expect { item.reload }.to_not raise_error
+            expect(response.status).to eq 401
```

Comment:
Well that is correct in this instance. This action can never be performed if a current_order_cycle is not present.